tests/run/cyfunction.pyx

```diff
@@ -409,3 +409,17 @@ def test_firstlineno_decorated_function():
     l1 = do_nothing.__code__.co_firstlineno
     l2 = test_firstlineno_decorated_function.__code__.co_firstlineno
     return l2 - l1
+
+def test_module():
+    """
+    >>> test_module.__module__
+    'cyfunction'
+    >>> type(test_module).__module__.startswith("_cython")
```

Comment:
This seems to be a bit fragile locally, because for some reason I always end up with an older cached cyfunction implementation.

It works when I manually import the module alone but not when I use runtests. I'm not sure what the CI will end up doing, but I'm fairly sure this is a caching problem and will go away when the shared lib version changes.